README-FR.md

```diff
@@ -0,0 +1,133 @@
+# if me
+
+  
+
+[if me](https://www.if-me.org/) est une communautée  sur les expériences sur la santé mentale qui encourage les gens a partager leur histoire personnel avec des personnes de confiance.
+les gens de confiance sont ceux avec qui on parle toues les jours , les membres de la famille, les amis, les professeurs, , et ceux qui travaille pour les expériences sur la santé.
```

Comment:
Should be capitalized to `Les`.

Reply:
Remove extra comma in `, et ceux qui travaille pour les expériences sur la santé`.